src/systemctl/systemctl.c

```diff
@@ -5963,8 +5964,17 @@ static int daemon_reload(int argc, char *argv[], void *userdata) {
 }
 
 static int trivial_method(int argc, char *argv[], void *userdata) {
+        static const char table[] =
+                "clear-jobs\0"   "ClearJobs\0"
```

Comment:
Why is this `char table[]` not `char *table`? Not that it matters much, since in both cases the compiler should put this in the const data section.

Reply:
This follows the recommendation from https://www.akkadia.org/drepper/dsohowto.pdf section 2.4.3 and later.

I mean, I wouldn't really do too much micro-optimization with such such tables, but where it is easy to use a nulstr over an strv I think, and things are within local scope I think it's still worth doing it.

Reply:
I don't think "optimization" matters here at all. The question for me is which syntax is more readable... The ternary-op approach is more localized.